src/components/MerchantsPage/gam/GiftAMealPage.tsx

```diff
@@ -34,43 +44,50 @@ const GiftAMealPage = (props: Props) => {
             </p>
           </div>
         </div>
-        <div className={styles.sub}>
-          <div className={styles.instructions}>
-            {[
-              ['Donate to Gift-a-Meal', gam_icon_step1],
-              [
-                '100% of donations still go directly to our merchants',
-                gam_icon_step2,
-              ],
-              [
-                'Vouchers are donated to our community parnters',
-                gam_icon_step3,
-              ],
-              ['Partners distribute to individuals in need', gam_icon_step4],
-            ].map(([text, icon], idx) => generateStep(idx + 1, text, icon))}
-          </div>
-        </div>
+        <CampaginInstructions isModal={false} />
       </div>
       <br />
       <button className={styles.backButton}>Back to merchants</button>
-      <h5 className={styles.campaignHeader}>Active Gift-a-Meal</h5>
-      // active campaigns here
+      {activeCampaigns.length ? (
+        <>
+          <h5 className={styles.campaignHeader}>Active Gift-a-Meal</h5>
+          {activeCampaigns.map((campaign: any) => (
+            // TODO: pass campaign data to CampaignListItem
+            <CampaignListItem campaign={''} />
+          ))}
+        </>
+      ) : (
+        <div className={styles.noCampaign}>
+          <div className={styles.noCampaignText}>
+            <p>No active campaigns? You can still donate!</p>
+            <p>
+              Meal donations here will automatically be applied to our next
+              campaign.
```

Comment:
Could you extract these strings into i18n?

Reply:
Yes! I'll convert all of the text this evening as well.